.github/workflows/e2e-tests-atomic.yml

```diff
@@ -0,0 +1,52 @@
+name: "E2E Tests on Atomic - All"
+
+# This action will run when it is triggered manually or on Pull Request with base=trunk
+on:
+  pull_request:
+    branches:
+      - trunk
+  workflow_dispatch:
+
+env:
+  WC_E2E_SCREENSHOTS: 1
+  E2E_SLACK_CHANNEL: ${{ secrets.E2E_SLACK_CHANNEL }}
+  E2E_SLACK_TOKEN: ${{ secrets.E2E_SLACK_TOKEN }}
+  E2E_WP_VERSION: 'nightly'
+  E2E_WC_VERSION: 'latest'
+  E2E_RESULT_FILEPATH: 'tests/e2e/results.json'
+  NODE_ENV: 'atomic'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  # Run tests against WP Nightly & WC latest. Only one combination at a time.
+  wp-nightly-tests:
+    name: "WP - nightly | WC - latest | ${{ matrix.test_groups }} - ${{ matrix.test_branches }}"
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        test_groups: [ 'wcpay', 'subscriptions', 'upe', 'upeSplit' ]
```

Comment:
Should we keep this matrix updated based on the ones we're planning to begin with instead of the `exclude` groups? 🤔 We can then add and update here once we enable more, not sure which approach seems better so starting this conversation.

Reply:
I don't have a preference.

I initially chose this approach to explicitly see what is currently missing compared to the regular E2E tests, but that's a detail